README.md

```diff
@@ -1,8 +1,14 @@
 Main Branch Status: ![CI/CD Workflow](https://github.com/Health-Education-England/tis-trainee-forms/workflows/CI/CD%20Workflow/badge.svg?branch=main)  
 Deployment Status: ![CI/CD Workflow](https://github.com/Health-Education-England/tis-trainee-forms/workflows/CI/CD%20Workflow/badge.svg?branch=main&event=deployment_status)
-# TIS Trainee Forms
+# TIS Trainee Forms Service
 
 ## About
+
+This service is used to retrieve and parse forms from an S3 Bucket, and to attach to saved forms
+the 'submissionDate' and 'lastModified' values before saving to the bucket.  Forms that are submitted
+are saved into the S3 bucket while those that are saved as draft are stored within a Mongo database
```

Comment:
I think maybe this is a bit clearer: 'Forms that are not in a Draft state are saved into the S3 bucket; all forms are saved in the Mongo database'